Webinject/Makefile.PL

```diff
@@ -24,6 +24,10 @@ requires 'URI'                     => 0;
 
 install_script 'bin/webinject.pl';
 
+if (!-f "README" || -M "lib/Webinject" < -M "README") {
```

Comment:
Generate README only if really necessary. Hopefully this would not be needed, but allows you to edit README before ship if you like. If not, I can just remove the `if`